serialisation/src/main/java/org/ehrbase/serialisation/xmlencoding/CanonicalXML.java

```diff
@@ -127,7 +127,10 @@ public <T extends RMObject> T unmarshal(String value, Class<T> clazz) {
         T composition;
         try {
             Unmarshaller unmarshaller = JAXBUtil.getArchieJAXBContext().createUnmarshaller();
-            // Set the parent XMLReader on the XMLFilter
+            unmarshaller.setAdapter(DateTimeXmlAdapter.class, new SdkDateTimeXmlAdapter());
```

Comment:
@vmueller-vg please if that make sense

Reply:
ArchieObjectMapperProvider.getObjectMapper() is used in different places. Do we need to add for all of them de/serializers or adapters you've created for date time?

CanonicalJson.unmarshalToMap
FlatJsonUnmarshaller.unmarshal
StdToCompositionWalker.handleRaw
etc.
@vmueller-vg